lib/question.js

```diff
@@ -325,8 +326,44 @@ module.exports = {
         debug('_saveSubmissionWithClient()');
         submission.raw_submitted_answer = submission.submitted_answer;
         submission.gradable = true;
+        submission.workspace = question.workspace_image !== null;
```

Comment:
```suggestion
        submission.workspace = question.workspace_image !== null;
```
This is not an actual `submission` column that ever gets synced to the DB, so maybe this is confusing usage?

Reply:
Do we even need this? Can't we just use `question.workspace_image != null`?